sixMans/game.py

```diff
@@ -52,7 +56,10 @@ async def _notify(self, new_state=None):
         if new_state:
             self.game_state = new_state
         for observer in self.observers:
-            await observer.update(self)
+            try:
+                await observer.update(self)
+            except:
+                pass
```

Comment:
I think throwing any exception here would cause the same issue we were trying to prevent with this try catch. In terms of logging the issue I'm not sure how to do that but I can put a TODO here to look into it later.